src/services/benefit/benefits.tsx

```
@@ -118,84 +118,76 @@ export const applyApplication = async ({
 	// }
 };
 interface ConfirmApplicationParams {
-	submission_id: string | undefined;
 	item_id: string | undefined;
-	benefit_id: string | undefined;
-
-	context: {
-		bpp_id?: string;
-		bap_uri?: string;
-	};
+	rawContext: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)